Update advanced-hunting-shared-queries.md (#3904)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/mtp/advanced-hunting-shared-queries.md

![Image of shared queries](../../media/advanced-hunting-shared-queries.png)

### Before you begin

> Users with one of the following levels of permissions to access Microsoft denfender.

Reviewer: ```suggestion
Users need one of the following levels of permissions to access Microsoft Defender:
```

> Full access (Read and Write)

Reviewer: ```suggestion
- Full access (read and write)
```

> Read-only access

Reviewer: ```suggestion
- Read-only access
```

> **Full access**

Reviewer: ```suggestion
**Full access**:
```

> Users with full access can Save,Modify and share a query. Assigning full access rights requires adding the users to the "Security Administrator" or "Global Administrator" AAD built-in roles.

Reviewer: ```suggestion
Users with full access can save, modify, and share a query. Assigning full access rights requires adding the users to the "Security Administrator" or "Global Administrator" AAD built-in roles.
```

> **Read-only access**

Reviewer: ```suggestion
**Read-only access**:
```

> Users with read-only access can log in, view all alerts, and related information. They will not be able to save, modify or share a query. Assigning read-only access rights requires adding the users to the "Security Reader" Azure AD built-in role.

Reviewer: ```suggestion
Users with read-only access can log in and view all alerts and related information. They will not be able to save, modify, or share a query. Assigning read-only access rights requires adding the users to the "Security Reader" Azure AD built-in role.
```
